test/correctness/compute_with.cpp

```diff
@@ -2110,6 +2110,93 @@ int rvar_bounds_test() {
     return 0;
 }
 
+int two_compute_at_test() {
```

Comment:
Worth commenting with the issue number that this test case came from, so that later readers have context